app/decorators/contribution_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class ContributionDecorator < Draper::Decorator
+  delegate_all
+
+  def created_at_formatted
```

Comment:
I believe this suffix `_formatted` is kinda redundant considering we're dealing with a decorator, I mean, attributes coming from this object are supposed to be decorated already.